artifact/rules.bzl

```diff
@@ -177,11 +198,7 @@ artifact_extractor = rule(
             mandatory = True,
             allow_single_file = True,
             doc = "Artifact archive to extract.",
-        ),
-        "strip_components": attr.int(
-            default = 2,
-            doc = "tar --strip-components argument (default 2).",
         )
     },
     executable = True,
-)
+)
```

Comment:
Missing newline at end of file!

Reply:
https://github.com/graknlabs/bazel-distribution/pull/253/commits/844aa511bfaf967c1cdf60880c3d2e8509321fb8